.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: required
```

Comment:
dtto jako logger, ale půjde to copy/paste, tak to snad nebude tolik práce

Reply:
mělo by to být